nh_observations/tests/nh_clinical_patient_observation/test_observations.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Open eObs. See LICENSE file for full copyright and licensing details.
-from openerp.tests import common
-from faker import Faker
 import logging
 
+from faker import Faker
```

Comment:
This test will most likely need refactoring if it's using faker (these tests will occassionally fail due to clashes with existing data)